File: exercises/ruby/flow_control/fizz_buzz/examples/004/fizz_buzz.rb
(1..100).each do |i|
  if i.modulo(15) == 0
    puts "FizzBuzz"
  elsif i.modulo(5) == 0
    puts "Buzz"
  elsif i.modulo(3) == 0
    puts "Fizz"
  else
    puts i
  end
end

File: exercises/javascript/flow_control/fizz_buzz/examples/001/fizz_buzz.js
function fizzbuzz(){
  numbers = [];
  for (var i=1;i<=100;i++){
    if (i % 15 == 0){
      numbers.push('FizzBuzz');
    } else if(i % 5 == 0){
      numbers.push('Buzz');
    } else if(i % 3 ==0){
      numbers.push('Fizz');
    } else {
      numbers.push(i);
    }
  }
  console.log(numbers);
}
